src/main/java/com/google/jenkins/plugins/computeengine/client/ComputeClient.java

```diff
@@ -337,10 +338,10 @@ public int compare(Subnetwork o1, Subnetwork o2) {
         return subnetworks;
     }
 
-    public Operation insertInstance(String projectId, Optional<String> template, Instance instance) throws IOException {
+    public Operation insertInstance(String projectId, String template, Instance instance) throws IOException {
         final Compute.Instances.Insert insert = compute.instances().insert(projectId, instance.getZone(), instance);
-        if (template.isPresent()) {
-            insert.setSourceInstanceTemplate(template.get());
+        if (StringUtils.isNotEmpty(template)) {
```

Comment:
Can you use !Strings.isNullOrEmpty(template) instead?

Reply:
example: https://github.com/jenkinsci/google-compute-engine-plugin/blob/653705bd813d86aa979e21a62461f206b8fd552b/src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java#L786